src/utils/index.ts

```diff
@@ -19,8 +19,12 @@ export function renderConfigToUrlParams(config: StacRenderObject): string {
   const searchParams = new URLSearchParams(queryObj);
 
   if (assets) {
-    for (let i = 0, len = assets.length; i < len; i++) {
-      searchParams.append('bands', assets[i]);
+    if (!params.asset_bidx) {
```

Comment:
This is a bit odd, but the presence of `asset_bidx` is the only way for us to establish whether we need to send `assets` as `bands` or not.